tools/tidb-binlog-cluster.md

```diff
@@ -392,6 +392,10 @@ The following part shows how to use Pump and Drainer based on the nodes above.
     
         # the address of the PD cluster nodes
         pd-urls = "http://192.168.0.16:2379,http://192.168.0.15:2379,http://192.168.0.14:2379"
+
+        # [storage]
+        # Set to true (by default) to ensure reliability and ensure binlog data is flushed to the disk
```

Comment:
change `and` to `by`? because it is not coordinating relation between `ensure reliability` and `ensure binlog data is flushed to the disk` , but `ensure binlog data is flushed to the disk` => `ensure reliability`